### PR TITLE
Feature/crypto byte finalized

### PR DESCRIPTION
**Pull request by mudasiri**
In this Pull Request I have built the following in line with the objectives of the project:

    1. Properly Documented the Project
    2. Used React components
    3. Used React props.
    4. Use React Router.
    5. Connected  React and Redux.
    6. Handle events in the app to navigate user to detail page and back.
    7. Wrote integration tests using React testing library.
    8. Used CSS styles in app.
    9. Used React life cycle methods like useEffect().
    10. Applied React best practices and language style guides in code.
    11. Used store, actions and reducers through SLicers.

A Loom recording of Project is [here](https://www.loom.com/share/37ab72b4ccd041fe849092378257bad8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,194 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## Learn More
+- [Live Demo Link](https://cryptobyte-react-app.onrender.com)
```
Comment (OLIPLICHE) [inline README.md:68]: - Hi, I would advise you to provide the proper Link to the deployed project. Here is what I am getting. Your ReadMe is the first contact between your project and a visitor. Therefore, leaving this link as it is will not show professionalism. 
Capture1

File: README.md [hunk 2]
```diff
@@ -1,70 +1,194 @@
...
+If you like this project give a 🌟.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for a good documentation.
+I would like to thank Nelson Sakwa for a good ui design on behance which was used as a UI guide for this project.
```
Comment (OLIPLICHE) [inline README.md]: - I would advise you to Link **Nelson Sakwa** to his website page. It is the best way to acknowledge the author of a website or App design.
- Example: [Nelson Sakwaa](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)),

File: src/App.css [hunk 3]
```diff
@@ -1,39 +1,77 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
```
Comment (OLIPLICHE) [inline src/App.css:2]: - [ ] **Design**:
- In reading the project requirement, I would advise you to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
The Layout: **composition and space between elements**.
Typography: font face, size, and weight.

- [ ] See your design: **Home page**
Capture

- [ ] The Original design: **Home and Detail**


Capture1

File: src/tests/__snapshots__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <Router>
+          <App />
+        </Router>
+        ,
+      </Provider>,
+    );
+    const testComponent = myRenderer.toJSON();
+    expect(testComponent).toMatchSnapshot();
+  });
+});
```
Comment (OLIPLICHE) [inline src/tests/__snapshots__/app.test.js:8]: - [ ] Testing: In reading the requirement of your Project, I noticed that your testing did not cover 100% of the requirements. Therefore, I would advise you to:
- to write the unit tests for several components of the app and some pure functions in Redux code.
- to write integration tests using React Testing Library. 

- [ ] I know you can make it, keep trying 💪 

Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @mudasiri, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @mudasiri, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* [x]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**